mmaction/datasets/pipelines/augmentations.py

```diff
@@ -697,6 +697,141 @@ def __repr__(self):
+            eigenvalues. Default: None.
+        eig_vec (np.ndarray | None): Eigenvectors of [3 x 3] size for RGB
+            channel jitter. If set to None, it will use the default
+            eigenvectors. Default: None.
+    """
+
+    def __init__(self,
+                 color_space_aug=False,
+                 alpha_std=0.1,
+                 eig_val=None,
+                 eig_vec=None):
+        if eig_val is None:
+            # note that the data range should be [0, 255]
+            self.eig_val = np.array([55.46, 4.794, 1.148])
+        else:
+            self.eig_val = eig_val
+
+        if eig_vec is None:
+            self.eig_vec = np.array([[-0.5675, 0.7192, 0.4009],
+                                     [-0.5808, -0.0045, -0.8140],
+                                     [-0.5836, -0.6948, 0.4203]])
+        else:
+            self.eig_vec = eig_vec
+
+        self.alpha_std = alpha_std
+        self.color_space_aug = color_space_aug
+
+    @staticmethod
+    def brightnetss(imgs, delta):
+        """Brightness distortion."""
+        if random.uniform(0, 1) > 0.5:
+            delta = np.array(delta).astype(np.float32)
+            imgs = imgs + delta
+        return imgs
+
+    @staticmethod
+    def contrast(imgs, alpha):
+        """Contrast distortion"""
+        if random.uniform(0, 1) > 0.5:
+            alpha = np.array(alpha).astype(np.float32)
+            imgs = imgs * alpha
+        return imgs
+
+    @staticmethod
+    def saturation(imgs, alpha):
+        """Saturation distortion."""
+        if random.uniform(0, 1) > 0.5:
+            gray = imgs * np.array([0.114, 0.587, 0.299]).astype(np.float32)
+            gray = np.sum(gray, 2, keepdims=True)
+            gray *= (1.0 - alpha)
+            imgs = imgs * alpha
+            imgs = imgs + gray
+        return imgs
+
+    @staticmethod
+    def hue(img, alpha):
+        """Hue distortion"""
+        if random.uniform(0, 1) > 0.5:
+            u = np.cos(alpha * np.pi)
+            w = np.sin(alpha * np.pi)
+            bt = np.array([[1.0, 0.0, 0.0], [0.0, u, -w], [0.0, w, u]])
+            tyiq = np.array([[0.299, 0.587, 0.114], [0.596, -0.274, -0.321],
+                             [0.211, -0.523, 0.311]])
+            ityiq = np.array([[1.0, 0.956, 0.621], [1.0, -0.272, -0.647],
+                              [1.0, -1.107, 1.705]])
+            t = np.dot(np.dot(ityiq, bt), tyiq).T
+            t = np.array(t).astype(np.float32)
+            # Here we use np.flip to suit our RGB images cases.
+            t = np.flip(t)
+            img = np.dot(img, t)
+        return img
+
+    def __call__(self, results):
+        imgs = results['imgs']
+        out = []
+        if self.color_space_aug:
+            bright_delta = np.random.uniform(-32, 32)
+            contrast_alpha = np.random.uniform(0.6, 1.4)
+            saturation_alpha = np.random.uniform(0.6, 1.4)
+            hue_alpha = random.uniform(-18, 18)
+            for img in imgs:
```

Comment:
so images in the same clip does not share the same jittering?

Reply:
It is more reasonable that images in the same clip share the same jittering. I add a `jitter_coin` variable to ensure that.